docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md

```diff
@@ -42,11 +42,10 @@ Relevant documentation for the components can also be found here:
 :::note Note on V3 Expectations for BigQuery
 
   A small number of V3 Expectations have not been migrated to BigQuery, and will be very soon. These include:
-  - `expect_column_values_to_be_in_set`
-  - `expect_column_values_to_be_in_type_list`
-  - `expect_column_values_to_be_between`
+
   - `expect_column_quantile_values_to_be_between`
-  - `expect_column_mean_to_be_between`
+  - `expect_column_kl_divergence_to_be_less_than`
```

Comment:
Edited this list to be more accurate. Now we are down to 2 Expectations that don't work yet in BigQuery only. This is excluding a number of statistical expectations that have not been migrated to V3